src/descriptor_runner/webdnn.ts

```diff
@@ -191,6 +192,18 @@ export async function load(directory: string, initOption: InitOption = {}): Prom
     backendOrder = backendOrder.slice();
     if (backendOrder.indexOf('fallback') === -1) backendOrder.concat(['fallback']);
 
+    // FIXME: User-defined transformDelegate is overridden
```

Comment:
Oh sorry..., you're right. I'll fix the document soon.  Also your idea looks good. Thank you.